identifiers/country-tw/township-level.csv

```diff
@@ -0,0 +1,70 @@
+id,name
+ocd-division/country:tw/special_municipality:tw-khh/district:constituency1,The first electoral district Kaohsiung City
+ocd-division/country:tw/special_municipality:tw-khh/district:constituency2,Kaohsiung second electoral district
```

Comment:
Instead of "district:constituencyN" how about "ed:N"

Reply:
What does ed type mean please ?

Reply:
ed is an abbreviation used for electoral district.